src/app/home/pages/ClustersPage/components/ClusterActionsDropdown.tsx

```diff
@@ -69,6 +74,7 @@ const ClusterActionsDropdown: React.FunctionComponent<IClusterActionsDropdownPro
                   onClick={() => {
                     setKebabIsOpen(false);
                     toggleAddEditTokenModal();
+                    setAssociatedCluster(clusterName);
```

Comment:
What's the reason for moving `associatedCluster` / `setAssociatedCluster` into redux and then using that in the modal instead of having the `preSelectedClusterName` prop passed to the `<AddEditTokenModal />` below?